README.md

```diff
@@ -0,0 +1,74 @@
+<h1 align="center" style="margin:1em;">
+  <a href="https://iris-grib.readthedocs.io/en/latest/">iris-grib</a>
```

Comment:
Why are you making the readme title an obfuscated link? If we want a link to the docs in the README, we should add it explicitly below.